### PR TITLE
Support StaticArray in loader

### PR DESCRIPTION
**Pull request by jtenner**

### CODE REVIEW

**Comment by jtenner** [comment]
Loader finally passed 🎉 .

This pull request contains fixes for:

- __allocArray()
- __getArray()

These methods now work with StaticArray types.

Should we add `__allocStaticArray()` and `__getStaticArray()` or leave everything as is?

File: lib/loader/tests/assembly/index.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/// <reference path="../../../../std/assembly/index.d.ts" />
```
Comment (dcodeIO) [inline lib/loader/tests/assembly/index.ts]: Suggesting to add a `tsconfig.json` to the directory instead

File: lib/loader/index.js [hunk 2]
```diff
@@ -233,30 +242,34 @@ function postInstantiate(extendedExports, instance) {
...
-  extendedExports.__getUint8ClampedArrayView = getTypedArrayView.bind(null, Uint8ClampedArray, 0);
-  extendedExports.__getInt16Array = getTypedArray.bind(null, Int16Array, 1);
-  extendedExports.__getInt16ArrayView = getTypedArrayView.bind(null, Int16Array, 1);
-  extendedExports.__getUint16Array = getTypedArray.bind(null, Uint16Array, 1);
-  extendedExports.__getUint16ArrayView = getTypedArrayView.bind(null, Uint16Array, 1);
-  extendedExports.__getInt32Array = getTypedArray.bind(null, Int32Array, 2);
-  extendedExports.__getInt32ArrayView = getTypedArrayView.bind(null, Int32Array, 2);
-  extendedExports.__getUint32Array = getTypedArray.bind(null, Uint32Array, 2);
-  extendedExports.__getUint32ArrayView = getTypedArrayView.bind(null, Uint32Array, 2);
+  const ctors = [
```
Comment (dcodeIO) [inline lib/loader/index.js]: This var and `bigctors` seem unnecessary, can just iterate over the literal

File: lib/loader/tests/assembly/index.ts (hunk 1 above)
Comment (jtenner) [reply lib/loader/tests/assembly/index.ts]: 👍

File: lib/loader/index.js (hunk 2 above)
Comment (jtenner) [reply lib/loader/index.js]: 👍

**Comment by dcodeIO** [comment]
Thank you! :)

**Pull request merged**